### ISSUE TITLE
This anti-adblock is not blocked

### ISSUE THREAD

**Issue by fallerallera** [reek/anti-adblock-killer#3733]
https://www.aftonbladet.se/sportbladet/fotboll/a/R3kjr/andersson-gladje-och-ilska-efter-triumfen
https://imgur.com/a/gXmum
Firefox 56 Linux
Adblock 3.6
Greasemonkey 3.17
Easylist + Swedish + Aaklist

**Comment by uBlock-user** [comment]
Can't reproduce.

@jspenguin2017

**Comment by jspenguin2017** [comment]
Can't reproduce

**Comment by Calle-L** [comment]
Yes, this anti-adblock pops up on every other page from Aftonbladet (Sweden's biggest newspaper), and I can confirm it happening on the page linked above.

What additional information might be needed for you to reproduce this problem?

My setup is:
Firefox 57.0.4 64 bit
Windows 10, 64 bit
Anti-adblock-killer v.10.0, Greasemonkey
Adblock Plus: AakList, Adblock Warning Removal List, EasyList, EasyPrivacy, Fanboy's Social Blocking List (no custom filters that apply)

[The same anti-adblock popups appear when using Chrome 63.0.3239.132 64 bit, with Adblock Plus, on this same computer]
